firmware/ec/src/main.c

```diff
@@ -9,6 +9,7 @@
 //*****************************************************************************
 //                                HEADER FILES
 //*****************************************************************************
+
```

Comment:
Yes, that was just done because linter only runs on changed files in Jenkins, so I needed a .c file to be be detected as changed so that the linter would be run. I will change it back with next commit.